MetaMask/metamask-extension
# Created review spending cap component

## Explanation

Created a review spending cup component. This component can't yet be added to the app. Instead, I added a storybook story that displays this component. This component received at least the following props:

1. tokenName (type: string)
2. currentTokenBalance (type: number)
3. tokenValue (type: number)
4. onEdit (type: function)

## More Information

* Fixes #15212 

## Screenshots/Screencaps


https://user-images.githubusercontent.com/92527393/185385264-26d9604c-f76f-4149-919a-66034606ec27.mov



## Manual Testing Steps

This component can be tested by running storybook (`yarn storybook`) and then go to the `COMPONENTS` tab and open in `UI/ReviewSpendingCup`. For testing this component you can see the Acceptance Criteria in the ticket: https://github.com/MetaMask/metamask-extension/issues/15212.
